css/style.css

```diff
@@ -0,0 +1,1114 @@
+  height: 50px;
+}
+
+.full-nav-social {
+  width: 100%;
+  height: 50px;
+  border-top: 1px solid #eeeeee50;
+  border-bottom: 1px solid #eeeeee50;
+}
+
+.full-nav-links {
+  display: flex;
+  width: 100%;
+  height: auto;
+  padding-top: 20px;
+}
+
+[class*="full-nav-col"] {
+  width: 50%;
+  height: auto;
+}
+
+.float-left {
+  float: left;
+}
+
+.float-right {
+  float: right;
+}
+
+.full-nav-social-wrapper {
+  position: relative;
+  display: inline;
+  margin-left: 20px;
+  top: 15px;
+}
+
+.full-nav-social-wrapper a {
+  margin: 0 6px;
+  text-decoration: none;
+  color: white;
+}
+
+.full-nav-logo {
+  height: 20px;
+  margin-top: 15px;
+  margin-left: 20px;
+}
+
+.full-nav-close {
+  position: relative;
+  margin-right: 20px;
+  top: 15px;
+  color: white;
+}
+
+.full-nav-col-1 {
+  display: block;
+  padding-left: 20px;
+  padding-right: 20px;
+  box-sizing: border-box;
+  font-size: 14px;
+  color: white;
+}
+
+.full-nav-col-1 a {
+  text-decoration: none;
+  color: rgba(255, 255, 255, 0.5);
+  font-size: 14px;
+  display: inherit;
+  margin: 10px 0;
+}
+
+.full-nav-col-2 {
+  display: block;
+  padding-left: 20px;
+  padding-right: 20px;
+  box-sizing: border-box;
+}
+
+.full-nav-col-2 a {
+  text-decoration: none;
+  color: rgba(255, 255, 255, 0.5);
+  font-size: 14px;
+  display: inherit;
+  margin: 10px 0;
+}
+
+[class*="full-nav-col"] h2 {
+  font-size: 16px;
+  color: white;
+}
+
+.header-wrapper {
+  height: 100%;
+  width: 100%;
+}
+
+.header-container-1-1 {
+  width: 65%;
+  height: 96%;
+  float: left;
+  background-color: lightseagreen;
+  background: url("https://cdn0.tnwcdn.com/wp-content/blogs.dir/1/files/2019/09/Ubisoft-Concordia-796x417.jpg");
+  background-repeat: no-repeat;
+  background-size: cover;
+  margin-top: 1%;
+  margin-bottom: 1%;
+}
+
+.header-container-1-2 {
+  height: 48%;
+  width: 34%;
+  margin-left: 66%;
+  background-color: lightgreen;
+  background: url("https://cdn0.tnwcdn.com/wp-content/blogs.dir/1/files/2018/09/maduro-petro-october-1st-cryptocurrency-national-oil-796x448.jpg");
+  background-repeat: no-repeat;
+  background-size: cover;
+  margin-bottom: .5%;
+  margin-top: 10px;
+}
+
+.header-container-2-2 {
+  height: 49%;
+  width: 34%;
+  margin-left: 66%;
+  background-color: violet;
+  background: url("https://cdn0.tnwcdn.com/wp-content/blogs.dir/1/files/2019/09/OnePlus7T-796x418.jpg");
+  background-repeat: no-repeat;
+  background-size: cover;
+  margin-top: .5%;
+}
+
+.gradient-cover {
+  position: relative;
+  width: 100%;
+  height: 100%;
+  background: url("https://cdn0.tnwcdn.com/wp-content/themes/cyberdelia/assets/img/cover-gradient.png");
+  opacity: .8;
+  background-repeat: no-repeat;
+  background-size: cover;
+  z-index : 99;
+}
+
+
+.fix-cover {
+  position: relative;
+  height: 40%;
+  margin-top: .5%;
+}
+
+.header-title-1 {
+  position: relative;
+  height: 50%;
+  width: 100%;
+  background: linear-gradient( 0deg, rgba(0, 0, 0, 1) 0%, rgba(0, 0, 0, 0) 100% ); 
```

Comment:
Can you try deleting the semicolon and adding it again (by typing it)?

I am thinking the semicolon is copied from somewhere and it's somehow invalid. :thinking: 

#### This is causing the Linter error.

### Do the same for #L203

Reply:
In fact yes, it was copied code from a css gradient tool, we deleted those semicolons and now it works perfectly TY!